.github/workflows/project-auto-add.yml

```diff
@@ -0,0 +1,94 @@
+# This workflow is used to add new issues to GitHub Kiota Project
+
+name: Add Issue or PR to project
+on:
+  issues:
+    types:
+      - opened
+  pull_request:
+    types:
+      - opened
+    branches:
+      - "main"
+
+jobs:
+  track_issue:
+    if: github.actor != 'dependabot[bot]' && github.event.pull_request.head.repo.fork == false
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a
+        with:
+          app_id: ${{ secrets.GRAPHBOT_APP_ID }}
```

Comment:
Any chance we can you double check the failing CI? Looks like this app_id may not be setup as this repo is in different organisation..